resources/api/reservation.py

```diff
@@ -441,7 +441,7 @@ def filter_is_favorite_resource(self, queryset, name, value):
         user = self.request.user
 
         if not user.is_authenticated:
-            return queryset.none if value else queryset
+            return queryset.none() if value else queryset
```

Comment:
It's a bit surprising to find this change in a PR which judging by the title contains only documentation updates.

If this is a bug found while exploring parameters for the documentation and fixed here, maybe create an issue and link it in the PR and/or update the PR title or description to reflect there's an actual code fix included.

Reply:
I created and issue and refered to it in the description.